person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] Kindy check on your parent_ permission, its suppose to be `parent_permission = true`, also ensure you initialize @rentals eg `@rentals = []` in your initialize method.

Reply:
Hello @danmainah, 
Actually with this syntax "parent_permission = true" I get linter errors